resources/compass/charts/connector/templates/role-binding.yaml

```diff
@@ -34,29 +34,29 @@ roleRef:
 {{ end }}
 
 
-{{ if .Values.global.connector.secrets.rootCA.name }}
+{{ if .Values.global.connector.secrets.rootCA.cacert }}
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
-  name: {{ template "fullname" . }}-{{ .Values.global.connector.secrets.rootCA.name }}
-  namespace: {{ .Values.global.connector.secrets.rootCA.namespace | default "default" }}
+  name: {{ template "fullname" . }}-{{ .Values.global.connector.secrets.rootCA.cacert }}
+  namespace: {{ .Values.global.connector.secrets.rootCA.cacertspace | default "default" }}
   labels:
     app: {{ .Chart.Name }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 rules:
 - apiGroups: ["*"]
   resources: ["secrets"]
-  resourceNames: ["{{ .Values.global.connector.secrets.rootCA.name }}"]
+  resourceNames: ["{{ .Values.global.connector.secrets.rootCA.cacert }}"]
   verbs: ["get"]
 
 ---
 kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: {{ template "fullname" . }}-{{ .Values.global.connector.secrets.rootCA.name }}
-  namespace: {{ .Values.global.connector.secrets.rootCA.namespace | default "default" }}
+  name: {{ template "fullname" . }}-{{ .Values.global.connector.secrets.rootCA.cacert }}
+  namespace: {{ .Values.global.connector.secrets.rootCA.cacertspace | default "default" }}
```

Comment:
I think it should still be `namespace`

Reply:
Yea definitely, got a little to wild with cmd+R xD